packages/calcite-components/src/components/stepper-item/stepper-item.tsx

```diff
@@ -80,33 +80,33 @@ export class StepperItem extends LitElement implements InteractiveComponent {
 
   //#region Public Properties
 
-  /** When `true`, the step has been completed. */
+  /** When present, the step has been completed. */
   @property({ reflect: true }) complete = false;
 
   /** A description for the component. Displays below the header text. */
   @property() description: string;
 
-  /** When `true`, interaction is prevented and the component is displayed with lower opacity. */
+  /** When present, interaction is prevented and the component is displayed with lower opacity. */
   @property({ reflect: true }) disabled = false;
 
-  /** When `true`, the component contains an error that requires resolution from the user. */
+  /** When present, the component contains an error that requires resolution from the user. */
   @property({ reflect: true }) error = false;
 
   /** The component header text. */
   @property() heading: string;
 
   /**
-   * When `true`, displays a status icon in the `calcite-stepper-item` heading inherited from parent `calcite-stepper`.
+   * When present, displays a status icon in the `calcite-stepper-item` heading inherited from parent `calcite-stepper`.
    *
    * @private
    */
   @property() icon = false;
 
-  /** When `true`, the icon will be flipped when the element direction is right-to-left (`"rtl"`). */
+  /** When present, the icon will be flipped when the element direction is right-to-left (`"rtl"`). */
   @property({ reflect: true }) iconFlipRtl = false;
 
   /**
-   * When `true`, the item will be hidden
+   * When present, the item will be hidden
```

Comment:
nitpick/potential follow-up PR: 

Add a period at the end of the sentence for consistency.

Reply:
I think so, either as part of this PR or a follow-up PR. I don't think we'd need an issue if it is included as part of a follow-up, think you could reference this PR in the follow-up PR for some reference.